docs/data_access.rst

```diff
@@ -66,80 +128,141 @@ Datasette
 ---------------------------------------------------------------------------------------
 
 We provide web-based access to the PUDL data via a
-`Datasette <https://datasette.io>`__ deployment at `<https://data.catalyst.coop>`__.
+`Datasette <https://datasette.io>`__ deployment at:
+
+  `<https://data.catalyst.coop>`__
+
+Datasette is an open source tool developed by
+`Simon Willison <https://https://simonwillison.net/>`__ that wraps SQLite databases in
+an interactive front-end. It allows users to the PUDL database and metadata, filter the
+data them using dropdown menus or SQL, and download the selected data to CSVs.  All the
+query parameters are stored in the URL so you can also share links to the data you've
+selected.
+
+The :doc:`PUDL data dictionary </data_dictionaries/pudl_db>` provides direct links to
+Datasette for each table if it is included in our SQLite outputs.
 
-Datasette is an open source tool that wraps SQLite databases in an interactive
-front-end. It allows users to browse database tables, select portions of them using
-dropdown menus, build their own SQL queries, and download data to CSVs. It also
-creates a REST API allowing the data in the database to be queried programmatically.
-All the query parameters are stored in the URL so you can also share links to the
-data you've selected.
+.. note::
+
+   The only SQLite database containing cleaned and integrated data is `the core PUDL
+   database <https://data.catalyst.coop/pudl>`__. There are also several FERC SQLite
+   databases derived from their old Visual FoxPro and new XBRL data formats, which we
+   publish as SQLite to improve accessibility of the raw inputs, but they should
+   generally not be used directly if the data you need has been integrated into the PUDL
+   database.
```

Comment:
another request to add a header link guy to the "Raw FERC DBF &...." section below and add a link to this in here